frontend/jupyter/src/app/pages/form/form-new/form-image/form-image.component.html

```diff
@@ -76,7 +76,7 @@
       >
         <mat-icon class="server-type-icon" svgIcon="group-three"></mat-icon>
         <p class="server-type-title">SAS</p>
-        <p class="server-type-content">This is text for SAS</p>
+        <p class="server-type-content" i18n>A SAS image.</p>
```

Comment:
Do we want a better name? Like A SAS image that allows to xyz?

Reply:
Better name, I don't think so. Better text description, yes. But I really don't know what text should be there.

Reply:
Yes description is what I meant. But at worst, it`s already better than what was there before.